gtsfm/two_view_estimator.py

```diff
@@ -25,21 +26,69 @@
 pil_logger = logging.getLogger("PIL")
 pil_logger.setLevel(logging.INFO)
 
+EPSILON = 1e-6
+
+
+
+@dataclass(frozen=False)
+class TwoViewEstimationReport:
+    """Information about verifier result on an edge between two nodes (i1,i2).
+
+    In the spirit of COLMAP's Report class:
+    https://github.com/colmap/colmap/blob/dev/src/optim/ransac.h#L82
+
+    Inlier ratio is defined in Heinly12eccv: https://www.cs.unc.edu/~jheinly/publications/eccv2012-heinly.pdf
+    or in Slide 59: https://www.cc.gatech.edu/~afb/classes/CS4495-Fall2014/slides/CS4495-Ransac.pdf
+
+    Args:
+        v_corr_idxs: verified correspondence indices.
+        num_inliers_est_model: #correspondences consistent with estimated model (not necessarily "correct")
```

Comment:
I think this should not be num_inliers. The results should be called num_verified.

Reply:
Thanks for the review, Ayush. I think `num_verified` is more confusing since it could refer to images or points or correspondences. I think `num_inliers` is more unambiguous in referring to correspondences

Reply:
@ayushbaid thanks for the review -- let me know what you think about my comment above when you get a chance

Reply:
I still think verified is better. But I am fine with inliers too as it is an established nomenclature.